controllers/certificatepolicy_controller.go

```diff
@@ -836,76 +708,6 @@ func (r *CertificatePolicyReconciler) sendComplianceEvent(ctx context.Context,
 	return r.Create(ctx, event)
 }
 
-func handleRemovingPolicy(name string) {
-	log.V(3).Info("Entered handleRemovingPolicy")
-
-	for k, v := range availablePolicies.PolicyMap {
-		if v.Name == name {
-			availablePolicies.RemoveObject(k)
-		}
-	}
-}
-
-func (r *CertificatePolicyReconciler) handleAddingPolicy(ctx context.Context, plc *policyv1.CertificatePolicy) {
-	log.V(3).Info("Entered handleAddingPolicy")
-
-	// clean up that policy from the availablePolicies list, in case the modification is in the
-	// namespace selector
-	for key, policy := range availablePolicies.PolicyMap {
-		if policy.Name == plc.Name {
-			availablePolicies.RemoveObject(key)
-		}
-	}
-
-	cleanupAvailablePolicies("", plc.Name)
-
-	addFlag := false
-
-	// Retrieve the namespaces based on filters in NamespaceSelector
-	selectedNamespaces := r.retrieveNamespaces(ctx, plc.Spec.NamespaceSelector)
-
-	for _, ns := range selectedNamespaces {
-		key := fmt.Sprintf("%s/%s", ns, plc.Name)
-		availablePolicies.AddObject(key, plc)
-
-		addFlag = true
-	}
-
-	if !addFlag {
-		key := fmt.Sprintf("/%s", plc.Name)
-		availablePolicies.AddObject(key, plc)
-	}
-}
-
-func cleanupAvailablePolicies(namespace string, name string) {
-	key := fmt.Sprintf("%s/%s", namespace, name)
-	if policy, found := availablePolicies.GetObject(key); found {
-		if policy.Name == name {
-			availablePolicies.RemoveObject(key)
-
-			if policy.Status.CompliancyDetails != nil {
-				delete(policy.Status.CompliancyDetails, namespace)
-			}
-		}
-	}
-}
-
-// =================================================================
-// Helper functions that pretty prints a map.
-func printMap(myMap map[string]*policyv1.CertificatePolicy) {
```

Comment:
Guessing since we don't need to keep things in sync we no longer need this print out.